.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Assign PRs
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Assign PR
+        uses: kentaro-m/auto-assign-action@v2.0.0
+        with:
+          reviewers: 'danielcdz'
+          assignees: 'danielcdz'
```

Comment:
I used to assign the PR to the author. Is it possible to automate this step, taking this into account?

Reply:
@danielcdz Yes, we can do that! Just one question do you still want to be assigned to the PR, or is it enough to add u as a reviewer only?

Reply:
Awesome! Just as reviewer is ok 🫡

Reply:
@danielcdz its updated now!